src/components/Authorize/Authorize.js

```diff
@@ -0,0 +1,27 @@
+import {useEffect} from 'react';
+import request from 'request';
+
+const Authorize = () => {
+  
+  useEffect(()=> {
+    let access_token = window.location.hash.substring(1).split('access_token=')[1].split('&')[0];
+    var options = {
+      url: 'https://api.spotify.com/v1/me',
+      headers: { 'Authorization': 'Bearer ' + access_token },
+      json: true
+    };
+
+        // use the access token to access the Spotify Web API
+    request.get(options, function(error, response, body) {
+      console.log(body);
+    });
+  }, []);
+
+  return (
+    <div>
+      <h1>Hello</h1>
+    </div>
+  )
```

Comment:
I think this should redirect to the `HomePage` when auth succeeds. Also we need error handling if auth fails.

Reply:
yes, i do this page for testing so on the console log you can see if everything it's working, with react context we can set the state of the authenitcation so we can redirect on the HomePage after login success